src/sentry/static/sentry/app/components/onboardingWizard/todoList.tsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import styled from '@emotion/styled';
 
-import getOnboardingTasks from 'app/components/onboardingWizard/getOnboardingTasks';
+import {getOnboardingTasks} from 'app/components/onboardingWizard/getOnboardingTasks';
```

Comment:
Why named over default?

Reply:
This exports a few different funs now. I'll rename the file

Reply:
File is now called `taskConfig`